docs/src/pages/system/the-sx-prop/PassingSxProp.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import ListItem from '@mui/material/ListItem';
+import FormLabel from '@mui/material/FormLabel';
+import { SxProps, Theme } from '@mui/material/styles';
+
+interface ListHeaderProps {
+  children: React.ReactNode;
+  sx?: SxProps<Theme>;
+}
+
+const ListHeader = ({ sx = [], children }: ListHeaderProps) => (
+  <ListItem
+    sx={[
+      {
+        width: 'auto',
+        textDecoration: 'underline',
+      },
+      // `SxProps` type can be an array, spread `sx` directly will give you type error.
```

Comment:
Let's add a comment here that otherwise there will be a type error? Or we can mention this [here](https://github.com/mui-org/material-ui/pull/29905/files#diff-993d43fb2174a87a11f8cf6c6aa9b7509821ef87f08f9945a632358d3fbb78b4R250)

Reply:
```suggestion
      // `SxProps` type can be an array, spreading `sx` directly will give you type error.
```

Reply:
It should be `// `SxProps` type can be an array. Spreading `sx` directly will give you type error.`